### PR TITLE
[Added][DD-79] Add support for employment records endpoint

### PR DESCRIPTION
**Pull request by blv-geoffreyvandercruyssen**
:question: What
---

Add support for the new Employment Records resource

:speech_balloon: Why
---

 
https://belvoteam.atlassian.net/browse/DD-79

### CODE REVIEW

File: tests/unit/test_employment_records.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    employment_records = resources.EmploymentRecords(api_session)
+    employment_records.session.post = MagicMock()
+    employment_records.create("fake-link-uuid", attach_pdf=False)
+
+    employment_records.session.post.assert_called_with(
+        "/api/employment-records/",
+        data={
+            "link": "fake-link-uuid",
+            "attach_pdf": False,
+            "save_data": False,
```
Comment (blv-albertmaristany) [inline tests/unit/test_employment_records.py:16]: Shouldn't `save_data` be `True` by default?

Comment (blv-geoffreyvandercruyssen) [reply tests/unit/test_employment_records.py:16]: I look at other resources and they had it as False. Do you want me to change the default to True?

Comment (blv-geoffreyvandercruyssen) [reply tests/unit/test_employment_records.py:16]: Btw the default is true, it is just for this test that it is false.

File: tests/unit/test_employment_records.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+from unittest.mock import MagicMock
+
+from belvo import resources
+
+
+def trest_employment_recrods_create(api_session):
```
Comment (blv-anisa) [inline tests/unit/test_employment_records.py:6]: nit
```suggestion
def test_employment_records_create(api_session):
```

File: tests/unit/test_employment_records.py (hunk 1 above)
Comment (blv-albertmaristany) [reply tests/unit/test_employment_records.py:16]: Maybe I got confused on what is the SDK request and what is being sent to the API.
Saw this:
```
employment_records.create("fake-link-uuid", attach_pdf=False)
```
And thought that the assertion would be `True` by default.

File: tests/unit/test_employment_records.py (hunk 2 above)
Comment (blv-geoffreyvandercruyssen) [reply tests/unit/test_employment_records.py]: Good spot!

File: tests/unit/test_employment_records.py (hunk 1 above)
Comment (blv-geoffreyvandercruyssen) [reply tests/unit/test_employment_records.py:16]: I see what you mean now and its weird that the test passed before....

**Review by blv-beni — APPROVED** [review]

**Comment by blv-geoffreyvandercruyssen** [comment]
@blv-albertmaristany Actually the test only started failing on the last commit, which is weird. So now I tested and it is working as it is supposed to work

**Pull request merged**